build/scripts/tests.sh

```diff
@@ -20,43 +20,38 @@ TARGET_FRAMEWORK=netcoreapp2.0
 XUNIT_CONSOLE_VERSION=$(get_package_version dotnet-xunit)
 XUNIT_CONSOLE=${NUGET_DIR}/dotnet-xunit/${XUNIT_CONSOLE_VERSION}/tools/${TARGET_FRAMEWORK}/xunit.console.dll
 
-echo Using $XUNIT_CONSOLE
+echo "Using ${XUNIT_CONSOLE}"
```

Comment:
is there a functionajl diff here or just preferred style

Reply:
Just style. (A lot of stuff in here is style)